test-common/utils.c

```diff
@@ -34,6 +35,10 @@
 #error "GPG_BINARY is not defined"
 #endif
 
+#ifndef FLATPAK_BINARY
+#error FLATPAK_BINARY is not defined
+#endif
```

Comment:
Since `FLATPAK_BINARY` is not used in this file, it seems a bit pointless to put this check here.